### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by faysal-alam-sazim**
### Description of Change
 - Created React+Vite with TypeScrip project
 - Configured Mantine UI library

### Associated Ticket Link
- https://trello.com/c/ng256CmC

### CODE REVIEW

**Comment by ragib-hossain-sazim** [comment]
Follow the sazim commit naming conventions for your commit. Look into Engineering doc for guideline.

**Comment by ragib-hossain-sazim** [comment]
For PR description, the sections doesn't have anything just write N/A instead of not adding them at all.

For e.g:

### Screenshots: N/A

File: src/assets/react.svg
Comment (ragib-hossain-sazim) [inline src/assets/react.svg:1]: Are these assets required for your TO-DO, if not then please remove them.

Comment (faysal-alam-sazim) [reply src/assets/react.svg:1]: Bhai, I've made changes to the last commit which is deleted unnecessary files.

**Comment by faysal-alam-sazim** [comment]
> For PR description, the sections doesn't have anything just write N/A instead of not adding them at all.
> 
> For e.g:
> 
> ### Screenshots: N/A

Updated PR description

**Review by ragib-hossain-sazim — APPROVED** [review]

**Pull request merged**